src/pages/settings/PaymentsPage.js

```diff
@@ -1,24 +1,109 @@
 import React from 'react';
-import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
-import Navigation from '../../libs/Navigation/Navigation';
+import {Pressable, TextInput, View} from 'react-native';
+import PropTypes from 'prop-types';
+import {withOnyx} from 'react-native-onyx';
+import CONST from '../../CONST';
+import ONYXKEYS from '../../ONYXKEYS';
 import ROUTES from '../../ROUTES';
+import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
+import Text from '../../components/Text';
 import ScreenWrapper from '../../components/ScreenWrapper';
+import NameValuePair from '../../libs/actions/NameValuePair';
+import {fetch} from '../../libs/actions/User';
+import Navigation from '../../libs/Navigation/Navigation';
+import styles from '../../styles/styles';
+
+const propTypes = {
+    // The PaypalMe Username
+    paypalMeAddress: PropTypes.string,
+};
+
+const defaultProps = {
+    paypalMeAddress: '',
+};
+
+class PaymentsPage extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            paypalMeAddress: props.paypalMeAddress,
+        };
+        this.setPaypalAccount = this.setPaypalAccount.bind(this);
+    }
+
+    componentDidMount() {
+        fetch();
+    }
+
+    componentDidUpdate(prevProps) {
+        if (prevProps.paypalMeAddress !== this.props.paypalMeAddress) {
+            // eslint-disable-next-line react/no-did-update-set-state
+            this.setState({paypalMeAddress: this.props.paypalMeAddress});
+        }
+    }
+
+    /**
+     * Sets the paypalMeAddress for the current user
+     */
+    setPaypalAccount() {
+        NameValuePair.set(CONST.NVP.PAYPAL_ME_ADDRESS, this.state.paypalMeAddress, ONYXKEYS.NVP_PAYPAL_ME_ADDRESS);
+    }
+
+    render() {
+        return (
+            <ScreenWrapper>
+                {() => (
+                    <>
+                        <HeaderWithCloseButton
+                            title="Payments"
+                            shouldShowBackButton
+                            onBackButtonPress={() => Navigation.navigate(ROUTES.SETTINGS)}
+                            onCloseButtonPress={() => Navigation.dismissModal()}
+                        />
+                        <View style={[styles.flex1, styles.p5]}>
+                            <View style={[styles.flex1]}>
+                                <Text style={[styles.textP, styles.mb4]}>
+                                    Enter your username to get paid back via Paypal.
+                                </Text>
+                                <Text style={[styles.formLabel]} numberOfLines={1}>
+                                    Paypal.me/
```

Comment:
Small thing - this should read `PayPal.me` with a second uppercase "P"

Reply:
@shawnborton In the issue, it was `Paypal.me/`. I have updated it now 😄